src/components/Sidebar/components/Backdrop/Backdrop.js

```diff
@@ -0,0 +1,33 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+const baseStyles = ({ theme, visible }) => css`
+  label: sidebar-backdrop;
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  background-color: black;
+  transition: opacity ${theme.transitions.default},
+    visibility ${theme.transitions.default};
+  visibility: ${visible ? 'visible' : 'hidden'};
+  opacity: ${visible ? 0.56 : 0};
```

Comment:
How did that magic number come about? 😅 Did the designers specify it? If this is a general number we want to stick with in different contexts, I would move it to the theme. If not, I would check with design, if we can use something we already have in the theme.

Reply:
It should only go into the theme, for now, if we actually use it in multiple places. `0.56` seems like such an arbitrary value, that it doesn't sound like the designer kept the overall design system in mind. We've used an opacity of `0.4` in other places whenever an element was disabled. That didn't go into the theme either, yet. I think it would be a good idea to have commonly shared opacity values. Maybe just bounce around the idea with the designer and see where you get. If you've already done that, just ignore my idea and keep the value where it is. 😉